index.html

```diff
@@ -10,43 +10,52 @@
 </head>
 
 <body>
-    <main class="main-section">
 
         <header>
             <nav>
+                <ul class="top-nav">
+                    <li>Portfolio</li>
+                    <li>About</li>
+                    <li>Contact</li>
+                </ul>
                 <img src="images/burgers.svg" alt="Menu-bar" class="hamburgers">
             </nav>
         </header>
+        <main class="main-section">
 
-        <section>
-            <div class="commence">
-                <h1 class="Introduction">
-
-                    Hey there. <br class="break">
-                    I'm Young. <br>
-                    <span class="green-h1">I'm a Software <br class="break">Developer</span>
-                </h1>
-                <div class="d-first-paragraph">
-                    <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
-                        my work
-                        and experience! If you like what you see and have a project you need coded,
-                        don’t hesitate to contact me.</p>
-                </div>
+        <!-- <section> -->
+        <div class="commence">
+            <h1 class="Introduction">
+
+                Hey there. <br class="break">
+                I'm Young. <br>
+                <span class="green-h1">I'm a Software <br class="break">Developer</span>
+
+            </h1>
+
+            <div class="d-first-paragraph">
+
+                <p class="first-paragraph"> I can help you build a product, feature or website Look through some of
+                    my work
```

Comment:
I would suggest you to use a section tag for each of the section page , The section tag is similar to the div tag, but it is more meaningful since it wraps logical groups of related content,kindly use the section tag instead of article for proper semantic reason and easy to read your code.